odl_video/envs.py

```diff
@@ -153,3 +153,24 @@ def get_key(name, default):
     if not isinstance(value, str):
         return value
     return value.encode().decode('unicode_escape').encode()
+
+
+def parse_env(env_file):
+    """
+    Parse the env file and set the values in the runtime environment.
+    Simplifies deployment by ensuring env values are present without
+    duplicating across multiple locations and not requiring explicit
+    sourcing.
+
+    :param env_file: path to a file consisting of key=value pairs
```

Comment:
@blarghmatey The typical format looks like [this](https://github.com/mitodl/odl-video-service/blob/master/odl_video/envs.py#L16-L22), can you change it? Otherwise looks good!